pkg/image/registry/imagestream/strategy.go

```diff
@@ -76,6 +79,16 @@ func (s Strategy) Validate(ctx kapi.Context, obj runtime.Object) field.ErrorList
 	}
 	errs := s.tagVerifier.Verify(nil, stream, user)
 	errs = append(errs, s.tagsChanged(nil, stream)...)
+
+	ns, ok := kapi.NamespaceFrom(ctx)
+	if !ok {
+		ns = stream.Namespace
+	}
+	err := s.limitVerifier.VerifyLimits(ns, stream)
+	if err != nil {
```

Comment:
You seem to be commenting some outdated commit.
On 05/26/2016 10:56 AM, Maciej Szulik wrote:
>
> In pkg/image/registry/imagestream/strategy.go <https://github.com/openshift/origin/pull/8195#discussion_r64711826>:
>
> > @@ -76,6 +79,16 @@ func (s Strategy) Validate(ctx kapi.Context, obj runtime.Object) field.ErrorList
> >  	}
> >  	errs := s.tagVerifier.Verify(nil, stream, user)
> >  	errs = append(errs, s.tagsChanged(nil, stream)...)
> > +
> > +	ns, ok := kapi.NamespaceFrom(ctx)
> > +	if !ok {
> > +		ns = stream.Namespace
> > +	}
> > +	err := s.limitVerifier.VerifyLimits(ns, stream)
> > +	if err != nil {
>
> This comment is still valid, you haven't changed it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/openshift/origin/pull/8195/files/9ad4b0157a8aceb5bb632ec6f9ccc38d912c563b..55e30d4f18bfee894db4a10cea5d3b0600ac6738#r64711826>
>